charts/seed-bootstrap/charts/nginx-ingress/templates/controller-service.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: nginx-ingress
+    component: "controller"
+  name: nginx-ingress-controller
+  namespace: garden
+spec:
+  ports:
+    - name: http
+      port: 80
+      protocol: TCP
+      targetPort: 80
+    - name: https
+      port: 443
+      protocol: TCP
+      targetPort: 443
+  selector:
+    app: nginx-ingress
+    component: "controller"
+    release: addons
+  type: "LoadBalancer"
```

Comment:
Good point, but out of scope of this PR. With this feature it will be possible to extend further managed ingress controller. For now, we only have nginx, but in the future we might want to add an istio ingress feature.

Reply:
Ok, sounds good to me